### PR TITLE
Removed int(32) and real(64) from ZarrTest

### PR DESCRIPTION
**Pull request by brandon-neth**
Some of the nightly are still failing due to ZarrTest, especially on the vm configurations. They're caused by a timeout. The ZarrTest file tests multiple datatypes, so I removed two of them so we're still testing both ints and reals and both 32 and 64 bit values. This should speed up the test and hopefully resolve the timeouts

Tested on chapvm and locally

Reviewed by:

### CODE REVIEW

**Review by ShreyasKhandekar — COMMENTED** [review]
What if we use `--fast`? I imagine that might speed us up enough to avoid the timeout without losing coverage?

File: test/library/packages/Zarr/ZarrTest.chpl [hunk 1]
```diff
@@ -185,7 +185,7 @@ proc updateChunkTest(type dtype) {
 proc main() {
   testGetLocalChunks();
   testUndistributedArray();
-  const typeTuple = (0:int(32), 0:int(64), 0:real(32), 0:real(64));
+  const typeTuple = (0:int(64), 0:real(32));
```
Comment (lydia-duncan) [inline test/library/packages/Zarr/ZarrTest.chpl:188]: Why remove them entirely, why not just separate them into another test file?

Comment (brandon-neth) [reply test/library/packages/Zarr/ZarrTest.chpl:188]: I can def break it into multiple test files; I'll probably break by testcase rather than type.

Comment (lydia-duncan) [reply test/library/packages/Zarr/ZarrTest.chpl]: Works for me :)

**Pull request merged**